sycl/source/device_selector.cpp

```diff
@@ -169,6 +168,10 @@ __SYCL_EXPORT int default_selector_v(const device &dev) {
   // The default selector doesn't reject any devices.
   int Score = 0;
 
+  if (dev.get_backend() == backend::ext_intel_esimd_emulator) {
```

Comment:
Could you please clarify if this is still required and if so, could you please add a comment describing this?